### DISCUSSION TITLE
How exactly is the project managed/organized?

### DISCUSSION THREAD

**Discussion by Scarfinos** [audacity/audacity#7492] [General]
I am currently working on a software ecology project focusing on an open-source project of my choice, and I decided to explore Audacity. However, I’m facing challenges in gathering detailed information about how the development process is organized. 

Specifically, I’m looking for answers to the following questions:
- Who reviews and approves pull requests?
- Who has the authority to make changes to the repository?
- How many full-time developers are dedicated to the project? What are their responsabilities?
- What role does the parent company, Muse Group, play in the development process?

I would greatly appreciate any insights or answers to even one of these questions. Thank you in advance for your help.

**Comment by LWinterberg** [comment]
Hello! It's easiest to answer these questions in reverse: 

> What role does the parent company, Muse Group, play in the development process?

All developers and contributors on the audacity team are Muse employees. There also are some community members which, while they generally don't submit as much code as team members for obvious reasons, provide invaluable insight into and support for certain areas (for example, accessibility, forum support and documentation).

> Are there any full-time developers dedicated to the project?

Yes.

> Who has the authority to make changes to the repository?  
> Who reviews and approves pull requests?

In principle any team member - after getting the PR approved by another team member (see [github branch protection](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule))

**Comment by Scarfinos** [comment]
I have other questions for my project:

- I noticed that some developers appear as collaborators not contributors. What is the difference?
- Who are the collaborators? What are their roles?
- How many collaborators/audacity team members are they?
- How many collaborators are working on the project?

**Reply by LWinterberg** [reply]
Contributors are people who have 1+ commit in the repo. Collaborators are people who have some permissions granted (eg being able to re-initiate a failed CI run). The difference isn't particularly meaningful, I believe most long term contributors are listed as collaborators. The team members we've had work on any particular release are listed in the credits in Help > About Audacity
